tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -56,147 +58,189 @@ std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>
 create_boundary_condition() {
   return std::make_unique<
       TestHelpers::domain::BoundaryConditions::TestBoundaryCondition<3>>(
+      // direction, blockID
       Direction<3>::upper_zeta(), 50);
 }
 
-auto make_domain_creator(const std::string& opt_string,
-                         const bool use_boundary_conditions) {
-  if (use_boundary_conditions) {
-    return TestHelpers::test_option_tag<
-        domain::OptionTags::DomainCreator<3>,
-        TestHelpers::domain::BoundaryConditions::
-            MetavariablesWithBoundaryConditions<3, domain::creators::Sphere>>(
-        opt_string + std::string{"  BoundaryCondition:\n"
-                                 "    TestBoundaryCondition:\n"
-                                 "      Direction: upper-zeta\n"
-                                 "      BlockId: 50\n"});
-  } else {
-    return TestHelpers::test_option_tag<
-        domain::OptionTags::DomainCreator<3>,
-        TestHelpers::domain::BoundaryConditions::
-            MetavariablesWithoutBoundaryConditions<3,
-                                                   domain::creators::Sphere>>(
-        opt_string);
-  }
-}
-
 template <typename... FuncsOfTime>
 void test_sphere_construction(
     const creators::Sphere& sphere, const double inner_radius,
     const double outer_radius, const double inner_cube_sphericity,
     const bool use_equiangular_map,
-    const std::array<size_t, 2>& expected_sphere_extents,
-    const std::vector<std::array<size_t, 3>>& expected_refinement_level,
-    const bool expect_boundary_conditions = false,
+    const std::vector<double> radial_partitioning = {},
```

Comment:
I think you can remove most arguments to this function once you deleted the map construction code below

Reply:
I removed `inner_cube_sphericity`, `use_equiangular_map`, & `expected_functions_of_time`.